samples/matter/light_bulb/CMakeLists.txt

```diff
@@ -53,6 +53,7 @@ target_sources(app PRIVATE
     ${COMMON_ROOT}/src/board/board.cpp
     ${COMMON_ROOT}/src/app/matter_init.cpp
     ${COMMON_ROOT}/src/app/matter_event_handler.cpp
+    ${COMMON_ROOT}/src/app/migration_manager.cpp
```

Comment:
This is used only for migration between 2.5.0 and 2.6.0, right? And if I develop an application from scratch based on 2.6.0 I don't need the migration. Considering this fact, I would not say we should use it by default in our samples. Maybe we should have another Kconfig to include it conditionally and inform user that this option should be enabled in the migration guide.